.gitignore

```diff
@@ -11,7 +11,6 @@ __diff_output__
 /.nyc_output
 /coverage
 /docs/.next
-/docs/pages/playground.tsx
 /docs/pages/playground
```

Comment:
Does `/docs/pages/playground` also catch `/docs/pages/playground.tsx` if it exists ?

Reply:
Did you mean `docs/pages/playground/playground.tsx`?
If no, then no, it doesn't.
Should we?
Are most devs used to experiment by editing `docs/pages/playground.tsx`?

Reply:
Sorry it was not clear
Does this new `.gitignore` correctly ignores  the old `/docs/pages/playground.tsx`

I still use it for my daily playground in which I just copy paste reproduction case, and `playground/new-pickers.tsx` (or others) for playground I want to keep for longer.

I could migrate to `playground/index.tsx` of course.
But we must be careful not to have someone push his old `playground.tsx` file by mistake.

Reply:
Ok, gotcha, fair point, I'll update the ignore to include a `*` then. 👍